internal/cmd/run/xcuitest.go

```diff
@@ -67,6 +67,8 @@ func NewXCUITestCmd() *cobra.Command {
 	sc.String("testAppDescription", "xcuitest::testAppDescription", "", "Specifies description for the testApp")
 	sc.StringSlice("otherApps", "xcuitest::otherApps", []string{}, "Specifies any additional apps that are installed alongside the main app")
 	sc.Int("passThreshold", "suite::passThreshold", 1, "The minimum number of successful attempts for a suite to be considered as 'passed'.")
+	sc.String("shard", "suite::shard", "", "When sharding is configured, saucectl automatically splits the tests by concurrency so that they can easily run in parallel.")
```

Comment:
Updated the description. Got a question for 

> This would be inconsistent with the other implementations and hard to extend.

Other frameworks don't provide this `shard` flag. Should I just remove it? Not sure about `hard to extend` 🤔